### ISSUE TITLE
Do not remove the block groups or block keys,etc...

### ISSUE THREAD

**Issue by chiqui3d** [sonata-project/SonataAdminBundle#2504]
Do not remove the block groups,this did not happen before,until I updated.
For example:

```
$formMapper->remove('groups')
```

This gives me the error:

```
Key "Security.Groups" for array with keys "User.Profile, User.General, User.Social, Security.Status, Security.Keys, Security.Roles, Gestion.Url y Foto" does not exist in SonataAdminBundle:CRUD:base_edit_form_macro.html.twig at line 4
```

**Comment by frak** [comment]
I can replicate this behaviour in 2.4.x-dev, not really sure where to start looking for the root cause for a patch...

**Comment by jordisala1991** [comment]
Should be fixed in #3773 Closing
